metagraph/CMakeLists.txt

```diff
@@ -493,14 +493,28 @@ add_dependencies(metagraph link_target)
 # install library
 #-------------------
 
-file(GLOB header_files
-  "src/*.hpp"
-  "src/*/*.hpp"
-  "src/*/*/*.hpp"
-)
+get_property(INCLUDE_DIRS DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
```

Comment:
My understanding is that this collects _everything_ specified in `include_directories`. I suppose on Mac it can as well collect `include_directories(SYSTEM /usr/local/include)`, which we probably do not want? I'm not sure what's the best course of action to avoid this situation here.